src/app/workflow/dag/dag.component.html

```diff
@@ -25,7 +25,7 @@
       <mat-card class="alert alert-warning" role="alert" *ngIf="!(dagResult$ | async) && !selectedVersion?.valid">
         <mat-icon>warning</mat-icon>&nbsp;
         <span *ngIf="(descriptorType$ | async) === ToolDescriptor.TypeEnum.WDL">
-          A Descriptor associated with this workflow could not be found.
+          Please ensure that the Descriptor is a valid WDL workflow.
```

Comment:
These two error messages, do they get displayed to both users and the author of the workflow?
Only the author actually has any ability to ensure that the WDL/CWL 1.0 workflow is valid.

Reply:
What happens if there is no descriptor associated with the workflow, as opposed to an invalid descriptor? Can it get in here, and if so, is that the message we want to display?

Also, it was like this already, but I don't think "Descriptor" should start with a capital.

Reply:
Not sure; I think maybe both.

Reply:
Definitely displayed to the author

Reply:
![Screen Shot 2020-02-11 at 9 28 16 AM](https://user-images.githubusercontent.com/3650921/74261945-db92af80-4cb0-11ea-8036-20728233a38c.png)

Reply:
So to elaborate, I think the message makes sense to show to the author. 
It makes less sense when shown to general users. 

Charles comment might be nice to tackle as well. i.e. the old message make sense if there are no descriptors, the new one makes sense if there are descriptors but they are invalid

Reply:
How about "The workflow descriptor either does not exist or is not valid WDL."

Reply:
Comment changes based on whether page is public or the My Workflows page